### PR TITLE
Outline code to handle emergency requests; must be merged with Dominik's code

### PR DESCRIPTION
**Pull request by samlowe106**

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -179,4 +179,56 @@ def get_contacts(request):
...
+    try:
+        phoneNumbers = map(lambda u: u.phoneNumber, UserContact.objects.filter(userId=userId))
+    except ObjectDoesNotExist:
+        phoneNumbers = []
+
+    message = _format_message(payload["message"])
+    _send_twilio_sms(message, userName, phoneNumbers)
+    responseMessageStrings.append("We alerted all of your emergency contacts.")
+
+    police_contacted = False
```
Comment (mattognibene) [inline api/views.py:217]: question: should both the alert authorities and alert emergency services share the same endpoint? if would just be a matter of whether or not we take a flag like `type=contacts` or `type=ems`. I have no strong preference, so whatever you think is best.

File: api/views.py [hunk 2]
```diff
@@ -179,4 +179,56 @@ def get_contacts(request):
...
+    return
+
+
+# handles the emergency request and sends emergency message user's contacts
+@api_view(["POST"])
+@csrf_exempt
+def handle_emergency_request(request):
+    payload = json.loads(request.body)
+    userId = payload["userId"]
+    userName = user_info.objects.get(id=userId)
```
Comment (mattognibene) [inline api/views.py:203]: should be `user_info.objects.get(id=userId).name`

**Review by mattognibene — APPROVED** [review]
Minor comments but looks solid

File: api/views.py (hunk 1 above)
Comment (samlowe106) [reply api/views.py:217]: Do you think we should ask the frontend team?

**Comment by samlowe106** [comment]
Merged

**Pull request merged**